frontend/javascripts/oxalis/model/actions/user_actions.ts

```diff
@@ -1,16 +1,17 @@
 import type { APIUser } from "types/api_flow_types";
-type SetActiveUser = {
-  type: "SET_ACTIVE_USER";
-  user: APIUser;
-};
-type LogoutUser = {
-  type: "LOGOUT_USER";
-};
+
+export const setActiveUserAction = (user: APIUser) =>
+  ({
+    type: "SET_ACTIVE_USER",
+    user,
+  } as const);
```

Comment:
`as const` is necessary so that the type is not interpreted as a string, but as a specific literal.